src/models/OAuthModel.php

```diff
@@ -125,20 +125,25 @@ public function createAccessTokenFromPassword($clientId, $username, $password)
      */
     protected function getUserId($username, $password)
     {
-        $sql  = 'SELECT ID, password, email FROM user
-            WHERE username=:username
-            AND verified = 1';
+        $sql  = 'SELECT ID, password, email, verified FROM user
+            WHERE username=:username';
         $stmt = $this->_db->prepare($sql);
         $stmt->execute(array("username" => $username));
```

Comment:
You're absolutely right! The ```if (! $result)``` should take care of that! Thanks for the eye-opener 👍